tests/redirects/map_globalconf.py

```diff
@@ -1373,4 +1373,7 @@
 
     # issue 9148
     url_test('/firefox/campaign/', '/firefox/new/'),
+
+    # Issue 9560
+    url_test('/openletter/', 'https://foundation.mozilla.org/blog/mozilla-urges-facebook-and-twitter-halt-dangerous-recommendations/?utm_source=mozilla.org&utm_content=shortlink'),
```

Comment:
Might want to split the query params out so that if they change order it doesn't break the tests.

```suggestion
    url_test('/openletter/', 'https://foundation.mozilla.org/blog/mozilla-urges-facebook-and-twitter-halt-dangerous-recommendations/', query={
    	'utm_source': 'mozilla.org',
    	'utm_content': 'shortlink',
    }),
```

Reply:
this will probably fix the flake8 complaint as well.

Reply:
Good call, and that also solves the long line flake8 was mad about